indicators/sledgehammerdiscordbot.yml

```diff
@@ -0,0 +1,32 @@
+title: sledgehammerfakeverification
```

Comment:
Titles should be unique, in case we have multiple different variations of the same thing.